fire/api/firedb.py

```diff
@@ -204,6 +205,59 @@ def tilknyt_landsnumre(
 
         return punktinfo
 
+    def fejlmeld_koordinat(self, sagsevent: Sagsevent, koordinat: Koordinat):
+        """
+        Fejlmeld en allerede eksisterende koordinat.
+
+        Hvis koordinaten er den eneste af sin slags på det tilknyttede punkt fejlmeldes
+        og afregistreres den. Hvis koordinaten indgår i en tidsserie sker en af to ting:
+
+        1. Hvis koordinaten forekommer midt i en tidsserie fejlmeldes den uden videre.
+        2. Hvis koordinaten er den seneste i tidsserien fejlmeldes den, den foregående
+           koordinat fejlmeldes og en ny koordinat indsættes med den foregåendes værdier.
+           Denne fremgangsmåde sikrer at der er en aktuel og gyldig koordinat, samt at
+           den samme koordinat ikke fremtræder to gange i en tidsserie.
+        """
+        punkt = koordinat.punkt
+
+        if len(punkt.koordinater) == 1:
+            self._luk_fikspunkregisterobjekt(koordinat, sagsevent, commit=False)
```

Comment:
I dette tilfælde er sagen vist OK: Der er kun en koordinat, og den er dårlig, men nedenfor glemmer du at checke for koordinatsystem: Hvis jeg  forstår korrekt, så kan man komme til at fejlmelde en DVR90  kote og erstatte den med en tidligere ETRS89 koordinat.

Reply:
Det har du ret i. Jeg har i min iver helt glemt den side af problemet!

Reply:
Det skal bare ikke være for nemt - men jeg tror ikke det kræver meget mere end et ekstra "if" at løse...